### PR TITLE
reusable PDF widget

### LINKED ISSUE #284: Update infographics for Version 1
@greggmiller Has some updated infographics which need integrating into the app for version 1.

### PR DESCRIPTION
**Pull request by rockgecko-development**
Implement #284 reusable multipage widget using a PDF viewer

Uses native_pdf_view lib.

## Changes
- Shows a 'Page x of y' subtitle
- Intubation Guide using Intubation_Guide_Infographics_Multipage.pdf from #284 
- Extubation Guide using Extubation_Guide_Infographics_Multipage_3rdMay.pdf from #284

## Screenshots
image

## Things to note

- I haven't deleted any of the png infographic files, which may now be redundant
- Replaces #302 which is now redundant

### CODE REVIEW

**Comment by rockgecko-development** [comment]
Also related: #252

**Comment by lukesleeman** [comment]
Again, I don't know much about the flutter side, but from testing out the app, both on an Android emulators (API 21 and 29!) and on a pixel 3, and S9, it seems to be working great!  pinch to zoom, and everything else is working really well.

Cant comment on the flutter side, but from the user perspective: 🚢 :shipit:

File: lib/view/reusable_infographic_pdf.dart [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                controller: pdfController,
+                scrollDirection: Axis.vertical,
+                onPageChanged: (page) {
+                  setState(() {
+                    _pageNumber = page;
+                  });
+                },
+                onDocumentLoaded: (_) => setState(() {
+                      _pageNumber = pdfController.page;
+                    }))));
```
Comment (maks) [inline lib/view/reusable_infographic_pdf.dart:72]: Could we please have the trailing commas pls 🙏

**Review by maks — APPROVED** [review]
Thanks @rockgecko-development 👍

**Comment by rockgecko-development** [comment]
why can't I merge?

**Review by workerbee22 — APPROVED** [review]
LGTM ... lookig at why you can't merge @rockgecko-development

**Pull request merged**